space_test.go

```diff
@@ -93,15 +93,43 @@ func (rest *TestSpaceREST) TestSuccessCreateSpace() {
 	assert.NotNil(t, created.Data.Links.Self)
 }
 
+func (rest *TestSpaceREST) TestSuccessCreateSpaceWithDescription() {
```

Comment:
I have a suggestion - could you please add a unit test for creating a space without a description?